index.html

```diff
@@ -14,119 +14,207 @@
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@400;700;800&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap"
+      rel="stylesheet"
+    />
     <meta name="theme-color" content="#e30062" />
   </head>
-
   <body>
-    <header>
-      <nav>
-        <!-- in case of small screens, show a dropdown -->
-        <input type="checkbox" id="nav-toggle" aria-controls="navbar toggle" />
-        <label for="nav-toggle" class="burger-menu"> &#9776;</label>
+    <div class="first-screen">
+      <header>
+        <a
+          href="https://www.realdevsquad.com/"
+          target="_blank"
+          class="logo-link"
+        >
+          <img
+            id="logo"
+            src="../img\re-designed images\RDS-logo.png"
+            alt="logo"
+          />
+        </a>
 
-        <ul class="menu">
-          <li>
-            <a class="nav-logo" href="https://realdevsquad.com/"
-              ><img
-                class="nav-logo-img"
-                src="./img/logos/Real-Dev-Squad@1x.png"
-                alt="Real Dev Squad logo"
-            /></a>
-            <a class="nav-home" href="https://realdevsquad.com/">Home</a>
-          </li>
-          <li>
-            <a class="active" href="https://welcome.realdevsquad.com/"
-              >Welcome</a
-            >
-          </li>
-          <li><a href="https://www.realdevsquad.com/events.html">Events</a></li>
-          <li><a href="https://members.realdevsquad.com/">Members</a></li>
-          <li><a href="https://crypto.realdevsquad.com/">Crypto</a></li>
-          <li><a href="https://status.realdevsquad.com/">Status</a></li>
-        </ul>
+        <div class="hamburger">
+          <div class="hamburger-lines"></div>
+          <div class="hamburger-lines"></div>
+          <div class="hamburger-lines"></div>
+        </div>
 
-        <div class="login-btn">
+        <nav class="main-nav">
           <a
-            href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+            class="nav-text welcome"
+            href="https://welcome.realdevsquad.com/"
+            target="_blank"
+            >Welcome</a
           >
-            Login
-          </a>
-        </div>
-      </nav>
-    </header>
-
-    <main>
-      <div class="container">
-        <h1 class="welcome-title">Welcome to Real Dev Squad!🎉</h1>
-
-        <div class="cards-title">
-          <h3>Check out our Docs ⬇️</h3>
-        </div>
-        <div class="cards">
-          <a class="card" href="./faq.html">FAQ</a>
-          <a class="card" href="./discord.html">Discord</a>
-          <a class="card" href="./code-of-conduct.html">Code of Conduct</a>
-        </div>
-
-        <p class="info-note social-handles">
           <a
-            href="https://instagram.com/realdevsquad"
+            class="nav-text events"
```

Comment:
I don't see `.events` class styled anywhere in CSS?

Reply:
it is true that .events has not been used, but it can be used while re-designing the events-site. So in my opinion we can keep it anyhow!
What say @swarajpure ?

Reply:
> but it can be used while re-designing the events-site

Didn't get what you meant here. Can you please explain?

Reply:
![image](https://user-images.githubusercontent.com/66074296/135883614-95077d5a-7f42-49fe-969e-0a4f59439cc6.png)

according to the styling done the welcome text is green as it represents the page that is currently opened and the rest of the texts are white so, in case we need to design other pages, we have to separately select the texts corresponding to the pages opened, so it would help then. @swarajpure

Reply:
got it, so should I remove the classes then? @swarajpure